gdrive_sync/management/commands/update_content_disposition.py

```diff
@@ -0,0 +1,74 @@
+"""
+We should update the content disposition for video downloads and transcript downloads to attachment so
+they are downloaded instead of opening in a browser
+"""
+import os
+
+import boto3
+import botocore
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from gdrive_sync.models import DriveFile
+from videos.constants import DESTINATION_ARCHIVE, VIDEO_DOWNLOAD_PATTERN
+from videos.models import Video, VideoFile
+
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def overwrite_s3_object(path):
+    """Replace s3 object with object with correct metadata"""
+
+    s3 = boto3.resource("s3")
+    bucket = settings.AWS_STORAGE_BUCKET_NAME
+    extra_args = {
+        "Metadata": {"ContentDisposition": "attachment"},
+        "MetadataDirective": "REPLACE",
+        "ACL": "public-read",
+    }
+
+    s3.meta.client.copy({"Bucket": bucket, "Key": path}, bucket, path, extra_args)
+
+
+class Command(BaseCommand):
+    """
+    Set video and pdf content disposition to attachment
+    """
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        """
+        Run the command
+        """
+        videos = Video.objects.all()
```

Comment:
I wonder how long this will take to process all videos and pdfs, do you think it will finish before timing out, or would a bunch of batched celery tasks be better?